integration-tests/spring-rabbitmq/src/test/java/org/apache/camel/quarkus/component/spring/rabbitmq/it/SpringRabbitmqTest.java

```diff
@@ -39,69 +54,350 @@ class SpringRabbitmqTest {
     private final static String ROUTING_KEY_POLLING = "pollingKey";
     private ConnectionFactory connectionFactory;
 
+    /**
+     * <a:href link=
+     * "https://camel.apache.org/components/4.4.x/spring-rabbitmq-component.html#_default_exchange_name">DEFAULT EXCHANGE
+     * NAME</a:href>
+     */
```

Comment:
Not sure if `<a:href>` is valid HTML / JavaDoc?

Maybe just use the plain URL or remove the comment entirely. Likewise for the other JavaDoc blocks on the other test headings.

Reply:
It should be `<a href` 

But I'll remove the links, as the  test methods could be easily linked to the chapters via its names

Reply:
I removed the comments